lib/pure/collections/tables.nim

```diff
@@ -434,7 +434,6 @@ proc getOrDefault*[A, B](t: Table[A, B], key: A, default: B): B =
     let a = {'a': 5, 'b': 9}.toTable
     doAssert a.getOrDefault('a', 99) == 5
     doAssert a.getOrDefault('z', 99) == 99
-  result = default(B)
```

Comment:
No. Instead the parameter `default: B` should be `def: B`.

Reply:
Did both, and did it for all parameters named `default` in tables (`OrderedTable` had the same bug). There are others across the codebase but they're less harmful and this PR being simpler is better.

Reply:
Yeah I know but it's a risk I'm willing to take.